ts/components/services/ContactPreferencesToggles/index.tsx

```diff
@@ -70,7 +70,7 @@ const ContactPreferencesToggle: React.FC<Props> = (props: Props) => {
     [serviceId, loadServicePreference]
   );
 
-  const nav = useNavigationContext();
+  const nav = useNavigation();
```

Comment:
By using this new hook from navigation it looks that isFocused information is not changing properly, so, as an example, if we activate a bonus starting from its special service detail screen, when we navigate back and come back correctly to the service detail screen the preferences won't reload


https://user-images.githubusercontent.com/3959405/159503527-4fd821fc-1432-43f1-a666-6b7cb15a0075.mp4

Reply:
[This](https://github.com/pagopa/io-app/blob/a63fcfd90078cdecdc5918dce8087b7aa64bc15a/ts/components/services/ContactPreferencesToggles/index.tsx#L77) is the hook i was mentioning